api/modules/food/zomato_response.py

```diff
@@ -0,0 +1,26 @@
+class ZomatoResponse(object):
+    """
+    Specifies the response to be sent for the zomato details.
+    """
+
+
+def __init__(self, *args, **kwargs):
+
+        self.restaurant_id = kwargs.get("id", -1)
+        self.restaurant_name = kwargs.get("name", "")
+        self.restaurant_url = kwargs.get("url", "#")
+        self.restaurant_latitude = kwargs.get("latitude", 0)
+        self.restaurant_longitude = kwargs.get("longitude", 0)
+        self.avg_cost_2 = kwargs.get("avg2", 0)
+        self.currency = kwargs.get("currency", "")
+        self.restaurant_image = kwargs.get("image", "")
+        self.aggregate_rating = kwargs.get("rating", 0)
+        self.votes = kwargs.get("votes", 0)
+        self.phone_numbers = kwargs.get("phone_numbers", 0)
```

Comment:
Remove `phone_numbers`
Earlier I thought, API will give us this information, but we will have to make another call to get this, you can go ahead and remove this